allennlp/common/tensor.py

```diff
@@ -123,3 +123,68 @@ def masked_softmax(vector, mask):
     else:
         # There is no mask, so we use the provided ``torch.nn.functional.softmax`` function.
         return torch.nn.functional.softmax(vector)
+
+
+def get_text_field_mask(text_field_tensors: Dict[str, torch.Tensor]) -> torch.Tensor:
```

Comment:
I don't exactly see the need for this to be a function - we are making a big assumption about the name of the token field. This also applies to the mask function I wrote above, btw. What higher order functionality were you imagining?